src/request_list.js

```diff
@@ -130,25 +114,26 @@ const ensureUniqueKeyValid = (uniqueKey) => {
  *   Setting `keepDuplicateUrls` to `true` will append an additional identifier to the `uniqueKey`
  *   of each request that does not already include a `uniqueKey`. Therefore, duplicate
  *   URLs will be kept in the list. It does not protect the user from having duplicates in user set
- *   `uniqueKey`s however. It is the user's responsibility to ensure uniqueness of their unique keys,
+ *   `uniqueKey`s however. It is the user's responsibility to ensure uniqueness of their unique keys
  *   if they wish to keep more than just a single copy in the `RequestList`.
  */
-class RequestList {
+export class RequestList {
     constructor(options = {}) {
         checkParamOrThrow(options, 'options', 'Object');
 
-        const { sources, persistStateKey, state, keepDuplicateUrls = false } = options;
+        const { sources, persistStateKey, stateKeyPrefix, state, keepDuplicateUrls = false } = options;
```

Comment:
Maybe for cleanness, I would do parameters differently: 
- I would have here `persistStateKey` and `persistSourcesKey` parameters that are optional. Maybe you don't want to persist anything.
- And in function `openRequestList` where user passes request list name I would compose these from its name and `STATE_PERSISTENCE_KEY` and `SOURCES_PERSISTENCE_KEY` constants and pass it to `RequestList` constructor

Reply:
Then we don't need to be introducing new variables as `this.stateKey`. Just call it the same as parameter `this. persistStateKey`, `this.persistSourcesKey`.

And we don't need even `this.shouldPersist`. Just use `this. persistStateKey`, `this.persistSourcesKey` independently on each other. If one is defined then use it for persistence.

Reply:
Actually I don't have any strong preference for any of the ways, so please just agree on one :)

Reply:
Thinking about it again, if `openRequestList` becomes the primary way of creating request lists, then I guess we can do it Marek's way and still be quite approachable.

Reply:
I just had a feeling that there is too much stuff - persistStateKey, stateKeyPrefix, this.shouldPersist, this.stateKey, this.sourcesKey,...

And we could simplify it to only 2 properties - persistStateKey, persistSourcesKey